.github/CODEOWNERS

```diff
@@ -1,75 +1 @@
-# required for merging a PR (depends on repository settings).
-#
-# For details about syntax, see:
-# https://help.github.com/en/articles/about-code-owners
-# But here are some important notes:
-#
-# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
-#   which can be everywhere.
-# - Multiple owners are supported.
-# - Either handle (e.g, @github_user or @github/team) or email can be used. Keep in mind,
-#   that handles might work better because they are more recognizable on GitHub,
-#   eyou can use them for mentioning unlike an email.
-# - The latest matching rule, if multiple, takes precedence.
-
-# CI
-/.github/ @paritytech/ci @paritytech/release-engineering
-/.gitlab-ci.yml @paritytech/ci
-/.gitlab/ @paritytech/ci
-
-# XCM
-/polkadot/xcm/ @paritytech/xcm
-
-# WASM executor, low-level client <-> WASM interface and other WASM-related code
-/substrate/client/allocator/ @koute
-/substrate/client/executor/ @koute
-/substrate/primitives/panic-handler/ @koute
-/substrate/primitives/runtime-interface/ @koute
-/substrate/primitives/wasm-interface/ @koute
-/substrate/utils/wasm-builder/ @koute
-
-# Systems-related bits and bobs on the client side
-/substrate/client/sysinfo/ @koute
-/substrate/client/tracing/ @koute
-
-# Documentation audit
-/substrate/primitives/runtime @paritytech/docs-audit
-/substrate/primitives/arithmetic @paritytech/docs-audit
-# /primitives/core (to be added later)
-# /primitives/io (to be added later)
-
-# FRAME
-/substrate/frame/ @paritytech/frame-coders @paritytech/docs-audit
-/substrate/frame/nfts/ @jsidorenko @paritytech/docs-audit
-/substrate/frame/state-trie-migration/ @paritytech/frame-coders @cheme
-/substrate/frame/uniques/ @jsidorenko @paritytech/docs-audit
-
-# GRANDPA, BABE, consensus stuff
-/substrate/client/consensus/babe/ @andresilva
-/substrate/client/consensus/grandpa/ @andresilva
-/substrate/client/consensus/pow/ @sorpaas
-/substrate/client/consensus/slots/ @andresilva
-/substrate/frame/babe/ @andresilva
-/substrate/frame/grandpa/ @andresilva
-/substrate/primitives/consensus/pow/ @sorpaas
-
-# BEEFY, MMR
-/substrate/frame/beefy/ @acatangiu
-/substrate/frame/beefy-mmr/ @acatangiu
-/substrate/frame/merkle-mountain-range/ @acatangiu
-/substrate/primitives/merkle-mountain-range/ @acatangiu
-
-# Contracts
-/substrate/frame/contracts/ @athei @paritytech/docs-audit
-
-# NPoS and election
-/substrate/frame/election-provider-multi-phase/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/election-provider-support/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/elections-phragmen/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/nomination-pools/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/staking/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/primitives/npos-elections/ @paritytech/staking-core @paritytech/docs-audit
+bridges/snowbridge/*.rs @snowfork/snowbridge-devs
```

Comment:
Does this need to start with `/`?

Reply:
I don't understand? 🤔

Reply:
Looking at the substrate examples they start with `/`
```
/bridges/snowbridge/*.rs @snowfork/snowbridge-devs
```

Reply:
I see if the path starts with a slash it means from the root of the project, which is more correct. Updated in https://github.com/Snowfork/polkadot-sdk/pull/110/commits/95f4611771c143a216f7ca488a9322a86ac3a6fd.